src/components/PageHeader.astro

```diff
@@ -0,0 +1,82 @@
+<header>
+  <div class="container flex fixed justify-center !max-w-full !p-0 z-10">
+    <nav class="flex flex-col h-10 w-screen bg-sky-500">
```

Comment:
We should have a page title in here on the left corner and navigation on the right

Reply:
Add page title